### PR TITLE
Fix part of #2417: Fix Audio test correctness

### PR DESCRIPTION
**Pull request by anandwana001**
## Explanation

Fix part of #2417 
previously test was passing with invalid data. 

All test passing in Robolectric
Screenshot 2021-01-07 at 15 07 56

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by rt4914** [comment]
Will pass this PR to @BenHenning as there are too many PRs to review right now. If you still need me to have a look, please assign back. Thanks.

File: app/src/sharedTest/java/org/oppia/android/app/player/audio/AudioFragmentTest.kt [hunk 1]
```diff
@@ -190,12 +194,13 @@ class AudioFragmentTest {
     }
   }
 
+  // TODO(#2417): Need a fake audio library to run this test on espresso
```
Comment (BenHenning) [inline app/src/sharedTest/java/org/oppia/android/app/player/audio/AudioFragmentTest.kt:204]: Should this be marked as RunOn Robolectric until the TODO can be resolved? Ditto below.

File: app/src/sharedTest/java/org/oppia/android/app/player/audio/AudioFragmentTest.kt [hunk 2]
```diff
@@ -306,27 +312,20 @@ class AudioFragmentTest {
...
-          coordinates[1] = screenY
-          return coordinates
-        }
-      },
-      Press.FINGER, /* inputDevice= */ 0, /* deviceState= */ 0
-    )
+  private fun setProgress(progress: Int): ViewAction {
+    return object : ViewAction {
+      override fun perform(uiController: UiController, view: View) {
+        (view as SeekBar).progress = progress
```
Comment (BenHenning) [inline app/src/sharedTest/java/org/oppia/android/app/player/audio/AudioFragmentTest.kt:327]: Does this work in Espresso?

**Review by BenHenning — CHANGES_REQUESTED** [review]
Thanks @anandwana001! Left some thoughts.

File: app/src/sharedTest/java/org/oppia/android/app/player/audio/AudioFragmentTest.kt (hunk 2 above)
Comment (anandwana001) [reply app/src/sharedTest/java/org/oppia/android/app/player/audio/AudioFragmentTest.kt:327]: Yes

File: app/src/sharedTest/java/org/oppia/android/app/player/audio/AudioFragmentTest.kt (hunk 1 above)
Comment (anandwana001) [reply app/src/sharedTest/java/org/oppia/android/app/player/audio/AudioFragmentTest.kt:204]: Done

**Review by BenHenning — APPROVED** [review]
Thanks @anandwana001!

**Comment by anandwana001** [comment]
@MaskedCarrot as this is merged, you can continue your work.

**Pull request merged**